tpm2/test/tpm2_test.go

```diff
@@ -1478,3 +1478,58 @@ func TestPlainImport(t *testing.T) {
 		t.Errorf("Got %X, expected %X", outBuff, inBuff)
 	}
 }
+
+func TestPolicyPCR(t *testing.T) {
+	rw := openTPM(t)
+	defer rw.Close()
+
+	sessHandle, _, err := StartAuthSession(rw, HandleNull, HandleNull, make([]byte, 16), nil, SessionPolicy, AlgNull, AlgSHA1)
+	if err != nil {
+		t.Fatalf("StartAuthSession failed: %v", err)
+	}
+	defer FlushContext(rw, sessHandle)
+
+	sel := PCRSelection{Hash: AlgSHA1, PCRs: []int{0, 1, 2, 3}}
+	pcrs, err := ReadPCRs(rw, sel)
+	if err != nil {
+		t.Fatalf("ReadPCRs failed: %s", err)
+	}
+
+	// From TPM 2.0 Part 1, Selecting Multiple PCR:
+	// "The list of selectors is processed in order. The selected PCR are
+	// concatenated, with the lowest numbered PCR in the first selector being
+	// the first in the list and the highest numbered PCR in the last selector
+	// being the last."
+	//
+	// Concatenate all selected PCR values before hashing. sel.PCRs is already
+	// sorted in ascending order.
+	var expectedVal []byte
+	for _, pcr := range sel.PCRs {
+		expectedVal = append(expectedVal, pcrs[pcr]...)
+	}
+	t.Logf("expectedVal=%x", expectedVal)
+
+	// Hash algorithm must match the one in StartAuthSession.
+	expectedDigest := sha1.Sum(expectedVal)
+	t.Logf("expectedDigest=%x", expectedDigest)
+
+	tests := []struct {
+		desc           string
+		expectedDigest []byte
+		wantErr        bool
+	}{
+		// PolicyPCR expects a *digest* of the PCR value, not the value itself.
+		// Make sure PolicyPCR with value actually fails.
+		{desc: "success: digest passed in expectedDigest", expectedDigest: expectedDigest[:]},
```

Comment:
This is Good, but subtest descriptions are usually `CamelCase`, just like test names.

Reply:
Disagree, subtest descriptions are arbitrary text and `t.Run` normalizes them using few simple substitutions.
We've used camelCase in a few places here to match const/func names.

I'll merge without fixing this, but happy to discuss further offline.

Reply:
Np